src/machine/memory/tlb/tlb.cpp

```diff
@@ -191,6 +194,20 @@ bool TLB::reverse_lookup(Address paddr, VirtualAddress &out_va) const {
     }
     return false;
 }
+
+void TLB::handle_control_signal(uint32_t ctrl_info) {
+    CSR::PrivilegeLevel priv = unpack_priv(ctrl_info);
```

Comment:
This should not be needed at the end. The `AccessMode` reaches TLB thorugh read and write.